lib/linguist/languages.yml

```
@@ -8388,6 +8388,13 @@ sed:
   ace_mode: text
   tm_scope: source.sed
   language_id: 847830017
+templ:
+  type: programming
```

Markup is more appropriate for this

```suggestion
  type: markup
```